cirq-web/.gitignore

```diff
@@ -0,0 +1,9 @@
+# Ignore node modules
+node_modules/
+
+# Pytest files
+.pytest_cache/
+
+# Extras
+utils/
```

Comment:
At first it was for my helper scripts and things like that, but after I saw the dev scripts you implemented in Cirq (like `check/`), I thought it may be helpful to have a similar folder that helps with things like updating the `webpack.config.js` file automatically, stuff like that. Thinking more about it though it may be better just to have users enter the data into the config.js file themselves (I gave instructions in the README already)...there's nothing complex enough that warrants script help yet. However, its something that's pretty cool and if we get to the point where we do find something complex enough I'd want to try and implement it. Knowing what I know now, it probably will need to go in the root directory of Cirq anyway and not in `cirq-web`.

TLDR: Moving it out and deleting from gitignore.